File: Ruby/Basic-Numbers/fizzbuzz.rb
def fizzbuzz(number)
  index = 1
  while index <= number
    if index % 5 == 0 && index % 3 == 0
      puts "FIZZBUZZ"
    elsif index % 3 == 0
      puts "FIZZ"
    elsif index % 5 == 0
      puts "BUZZ"
    end
    index += 1
  end
end

fizzbuzz(15)

File: JavaScript/Basic-Numbers/fizzbuzz.js
function fizzbuzz(number) {
  for (let i = 1; i <= number; i++) {
    if (i % 3 === 0 && i % 5 === 0) {
      console.log("FIZZBUZZ");
    } else if (i % 3 === 0) {
      console.log("FIZZ");
    } else if (i % 5 === 0) {
      console.log("BUZZ");
    }
  }
}
console.log(fizzbuzz(15));